### DISCUSSION TITLE
Is  "Disable modern search bar" working?

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#669] [Q&A] [answered]
Am I missing something here or is this not working, I assume it should be replaced with the "classic search box" as discussed [here](https://www.reddit.com/r/Windows11/comments/pu5aa3/howto_disable_new_context_menu_explorer_command/)?

Toggling "Disable modern search bar" in Settings has no visible effect for me even after a restart of file explorer or a system reboot.

`HKCU\Software\Classes\CLSID\{1d64637d-31e9-4b06-9124-e83fb178ac6e}`

is modified correctly but with modern search bar enabled or disabled my File Explorer search bar always looks like this:

![Search bar](https://user-images.githubusercontent.com/48499740/149522400-97d10ae3-bdd3-4159-a016-691110209e92.png)

Tested on latest pre-release ep 22000.434.41.6_69707a3 and on my regular win 11 installation and a cleaner one both current at 22000.434.

**Comment by valinet** [comment]
I keep it disabled but yeah, for me it works. I am on the latest pre-release (it doesn't matter anyway) and latest stable Windows 11 build, 22000.434.

**Reply by chip33** [reply]
hmmm, I can't work out why it wouldnt work for me.
Could you do a screenshot of how the searchbar should look when disabled? 
Any idea how I could troubleshoot and fix this?
All I can think of is Im on windows 11 updated from windows 10 rather than a totally clean install but can't see how that should matter.
If I can't fix it I will do a clean install of win 11 on VM and try it.
Thanks

**Reply by valinet** [reply]
> Could you do a screenshot of how the searchbar should look when disabled?

![image](https://user-images.githubusercontent.com/6503598/149550613-e5a79f20-476b-426b-b81e-638d7e288246.png)

> Any idea how I could troubleshoot and fix this?

The way that works is by just having COM treat the component of the new search bar as the old one. Other than that, nothing else is really done in `explorer`. You could, you know, just forget about it, hide it completely altogether (that's what I do) and use Everything Search which is a million times better and faster anyways.

**Reply by chip33** [reply]
yes, I have Everything search alpha 1.5a with the dark mode, even in alpha it is excellent.
I read you are thinking of swapping windows search out for Everything if you get the time.

**Reply by valinet** [reply]
Yeah, I looked into that some time ago, but idk, maybe one day...

**Reply by chip33** [reply]
yes, big job I imagine.

**Reply by valinet** [reply]
On the surface, not looking that big, but in reality, everything computer releated takes insane amounts of time to do. And yeah, it is actually kind of big...

**Reply by chip33** [reply]
Yeah, I know, even the low level powershell and vbs scripts I do take me ages to get right. Very satisfying once you get them working though.

**Reply by chip33** [reply]
Here are some tweaks to add context menus to quickly Search Offline with Everything for a selected file or folder name or clipboard contents or search for anything within the open directory by clicking on directory background.
And to Search Online with chrome beta (mod it for any browser) for the same selected items.

[EverythingTweaks.zip](https://github.com/valinet/ExplorerPatcher/files/7872712/EverythingTweaks.zip)




Pretty handy for me.
I have renamed the .exe files to encourage anyone to read the included ahk , etc files and compile them into their own  exes to be sure there is no malicious code in there.

![Everything tweaks](https://user-images.githubusercontent.com/48499740/149557812-32045398-90b9-4c15-8606-194a2148327d.png)

**Reply by chip33** [reply]
edit: tidied up the files in the zip.

**Comment by chip33** [comment, marked as answer]
The solution to the Modern search bar always being disabled was due to just one registry setting as referenced[ here](https://www.thewindowsclub.com/enable-legacy-search-box-in-file-explorer-of-windows-10).

The only value I needed to change was the User Account Control  setting (which also disables  Modern search bar).
```
[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System]
"EnableLUA"=dword:00000000
```
 to
`"EnableLUA"=dword:00000001`

which enables it after a reboot.

I changed just this setting back and forth a few times and rebooted  just to confirm this as the solution.

**Reply by valinet** [reply]
Yeah, I mean, EnableLUA is known to have side effects. This turns out to be one of them. Leave that to the default setting.

**Reply by chip33** [reply]
Yes, it was an interesting solution.
